protocol/x/ratelimit/module_test.go

```diff
@@ -107,7 +128,10 @@ func TestAppModuleBasic_GetTxCmd(t *testing.T) {
 }
 
 func TestAppModuleBasic_GetQueryCmd(t *testing.T) {
-	createAppModuleBasic(t)
+	am := createAppModuleBasic(t)
 
-	// TODO(CORE-823): implement query for `x/ratelimit`
+	cmd := am.GetQueryCmd()
+	require.Equal(t, "vest", cmd.Use)
```

Comment:
nit:
```suggestion
	require.Equal(t, "ratelimit", cmd.Use)
```
below assertions need to be updated accordingly too